crates/ruff_python_formatter/tests/snapshots/format@statement__function.py.snap

```diff
@@ -996,4 +996,167 @@ def default_arg_comments2(  #
 ```
 
 
+## Preview changes
```

Comment:
I wonder how noisy this gets once we start having many preview style rules. But let's see how it goes.